Source/WebCore/platform/adwaita/ScrollbarThemeAdwaita.cpp

```diff
@@ -77,6 +80,8 @@ bool ScrollbarThemeAdwaita::usesOverlayScrollbars() const
 {
 #if PLATFORM(GTK)
     return shouldUseOverlayScrollbars();
```

Comment:
I think we can move the env var check here, and then just use the same code for GTK and WPE